app/views/index.scala.html

```diff
@@ -33,12 +33,10 @@
                     <br>
                     <a class="bodyStartBtn" href="@routes.AuditController.audit()">Start Exploring Seattle</a>
```

Comment:
Oh this is the culprit. The city name is hard-coded.

Reply:
yep